lib/web_ui/lib/src/engine/canvaskit/font_fallbacks.dart

```diff
@@ -17,6 +17,31 @@ import 'fonts.dart';
 import 'initialization.dart';
 import 'interval_tree.dart';
 
+
+// Runtime parameters.
+
+/// The URL to use when downloading the font resources.
+/// 
+/// The base URL can be overridden using the `FONT_API_SERVER`
+/// environment variable.
+///
+/// When specifying using the environment variable set it in the Flutter tool
+/// using the `--dart-define` option. The value must end with a `/`.
+///
+/// Example:
+///
+/// ```
+/// flutter run \
+///   -d chrome \
+///   --web-renderer=canvaskit \
+///   --dart-define=FONT_API_SERVER=https://example.com/
+/// ```
+const String defaultFontServerUrl = String.fromEnvironment(
```

Comment:
I would probably call this "fontServerUrl" since it can't be overridden so it's not the default. (The default is the one under `defaultValue` below.)

Reply:
maybe `fontServerBaseUrl`, since it's then concatenated with a path?

Reply:
done.